### ISSUE TITLE
Any way to prevent the duplication of slides?

### ISSUE THREAD

**Issue by ch-tactica** [ganlanyuan/tiny-slider#85]
My slider is duplicating a slide which has a form in it which is causing some issues for me. Is there any way to prevent this? I have loop set to false:

```javascript
const registrationSlider = tns({
  container: '.form-container',
  items: 1,
  slideBy: 'page',
  touch: false,
  mode: 'gallery',
  controls: false,
  loop: false,
  autoplay: false
});
```

**Comment by ganlanyuan** [comment]
Since you're using `gallery`, there's no way to get rid of it for now.
Maybe later I will remove the duplicate slides for `gallery`.

**Comment by JBriesen** [comment]
how can i get rid of the extra slides when using carousel mode?
Great slider btw

**Comment by ganlanyuan** [comment]
Hey @JBriesen ,
For `carousel` mode, you have to set `loop: false` and without `edgePadding`.

**Comment by lbineau** [comment]
Any news about this for carousel mode ?

**Comment by ganlanyuan** [comment]
Hey @lbineau you can use `loop: false` for now.

**Comment by ganlanyuan** [comment]
That's because you're using `gallery` mode. [refer to here](#issuecomment-334590999)

**Comment by lbineau** [comment]
Yeah that's why I ask if there is any news for this issue in carousel mode 😂
